src/Mesh.cc

```diff
@@ -25,59 +27,53 @@ Mesh::Mesh(FILE *input_file) {
       c = fread((void *)buffer, sizeof(float), 3 * 1024, input_file);
     }
     for (int i = 1; i < 3; i++) {
-      cout << "v:" << buffer[i] << std::endl;
+      std::cout << "v:" << buffer[i] << std::endl;
     }
+    assert(0 == 1 && "NIKOS: this is definitelly wrong, FIX IT");
```

Comment:
assert? I suppose this is not running now.

Reply:
No it should crash at run time. Reading from file is broken as the values read from the file are not actually stored in the mesh class. Am I wrong so far?

I did not try to fix it since this is not required and I wanted to focus on drawing the points on the screen and then the triangulation, but I added this "assertion" as a precautionary safety net.